.travis.yml

```diff
@@ -22,18 +17,29 @@ env:
   - MATRIX_TOXENV=integration-dynamodb
 matrix:
   include:
+  - python: '2.7'
+  - python: '3.4'
+  - python: '3.5'
   - python: '3.6'
+  #- python: '3.7'
+  #  dist: xenial    # required for Python 3.7 (travis-ci/travis-ci#9069)
+  - python: '3.7'
+    dist: xenial    # required for Python 3.7 (travis-ci/travis-ci#9069)
     env: TOXENV=pypy-unit PYPY_VERSION="pypy2.7-6.0.0"
```

Comment:
Can you explain why the PYPY test need Python 3.7 as a default system Python? Shouldn't it only be concerned with the PYPY version?

Reply:
My understand was that this was an integration tests between systems running two different versions of Python (CPython 3.7 and Pypy 2.7).  If that is not the case then __python: pypy__ should be the setting because in that case, we do not need CPython.